docs/Guidelines.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+The purpose of *this* document is to serve as a primer for developers/qe who are looking for best practices or add tests to RHTAP E2E framework.
```

Comment:
I would change "or add tests" to "or adding new tests" or "or to add new tests".